src/openrct2/ride/RideData.cpp

```diff
@@ -1958,7 +1958,7 @@ const rct_ride_properties RideProperties[RIDE_TYPE_COUNT] = {
          RIDE_TYPE_FLAG_HAS_LOAD_OPTIONS | RIDE_TYPE_FLAG_PEEP_WILL_RIDE_AGAIN | RIDE_TYPE_FLAG_HAS_VEHICLE_COLOURS | RIDE_TYPE_FLAG_CHECK_FOR_STALLING |
          RIDE_TYPE_FLAG_HAS_TRACK | RIDE_TYPE_FLAG_SUPPORTS_MULTIPLE_TRACK_COLOUR,
                 7,  30,  30, 40, 40, 0 },  // RIDE_TYPE_REVERSE_FREEFALL_COASTER
-        {RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_MAIN | RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_SUPPORTS | RIDE_TYPE_FLAG_3 |
+        {RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_MAIN | RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_SUPPORTS | RIDE_TYPE_FLAG_3 | /*RIDE_TYPE_FLAG_CAN_SYNCHRONISE_ADJACENT_STATIONS| DOES NOT WORK!*/
```

Comment:
Why does it not work? Is it a limitation that you didn't find or should it not work?

Reply:
I've tested it it. Yes the checkbox shows up on the GUI, but the towers do no sync actually. I assume that the syncing logic only applies to regular tracked rides.

Reply:
It does not work because it needs a STATION_END piece for that.

I don't think commenting it out will really help anyway, so I'd get rid of this comment altogether.